### PR TITLE
Fixing Title tags in metadata

### PR DESCRIPTION
**Pull request by juanpablojofre**
Version(s) of document impacted
------------------------------
- All versions were affected and all were reviewed independently of the others.

@zjalexander , please review and merge (once validations pass).

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.gitignore](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/.gitignore) | :white_check_mark:Succeeded | |
[reference/3.0/ISE/Get-IseSnippet.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/ISE/Get-IseSnippet.md) | :white_check_mark:Succeeded | |
[reference/3.0/ISE/Import-IseSnippet.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/ISE/Import-IseSnippet.md) | :white_check_mark:Succeeded | |
[reference/3.0/ISE/New-IseSnippet.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/ISE/New-IseSnippet.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Add-History.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Add-History.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Add-PSSnapin.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Add-PSSnapin.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Clear-History.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Clear-History.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Connect-PSSession.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Connect-PSSession.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Disable-PSRemoting.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Disable-PSRemoting.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Disable-PSSessionConfiguration.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Disable-PSSessionConfiguration.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Disconnect-PSSession.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Disconnect-PSSession.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Enable-PSRemoting.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Enable-PSRemoting.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Enable-PSSessionConfiguration.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Enable-PSSessionConfiguration.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Enter-PSSession.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Enter-PSSession.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Exit-PSSession.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Exit-PSSession.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Export-Console.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Export-Console.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Export-ModuleMember.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Export-ModuleMember.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/ForEach-Object.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/ForEach-Object.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Functions/Clear-Host.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Functions/Clear-Host.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Functions/Get-Verb.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Functions/Get-Verb.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-Command.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-Command.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-Help.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-Help.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-History.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-History.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-Job.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-Job.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-Module.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-Module.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-PSSession.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-PSSession.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-PSSessionConfiguration.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-PSSessionConfiguration.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Get-PSSnapin.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Get-PSSnapin.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Import-Module.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Import-Module.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Invoke-History.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Invoke-History.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/New-Module.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/New-Module.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/New-ModuleManifest.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/New-ModuleManifest.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/New-PSSession.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/New-PSSession.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/New-PSSessionConfigurationFile.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/New-PSSessionConfigurationFile.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/New-PSSessionOption.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/New-PSSessionOption.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/New-PSTransportOption.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/New-PSTransportOption.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Out-Default.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Out-Default.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Out-Host.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Out-Host.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Out-Null.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Out-Null.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/Alias-Provider.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/Alias-Provider.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/Environment-Provider.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/Environment-Provider.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Add-Content-for-FileSystem.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Add-Content-for-FileSystem.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Clear-Content-for-FileSystem.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Clear-Content-for-FileSystem.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Get-ChildItem-for-FileSystem.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Get-ChildItem-for-FileSystem.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Get-Content-for-FileSystem.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Get-Content-for-FileSystem.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Get-Item-for-FileSystem.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Get-Item-for-FileSystem.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Remove-Item-for-FileSystem.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Remove-Item-for-FileSystem.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Set-Content-for-FileSystem.md](https://github.com/PowerShell/PowerShell-Docs/blob/jpjofre-I1188/reference/3.0/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Set-Content-for-FileSystem.md) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C3%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201705031751079363-1190%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/ISE/Import-IseSnippet.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Import-IseSnippet [-Recurse] -Module <String> [-ListAvailable] [<CommonParameter
 ## DESCRIPTION
 The **Import-IseSnippet** cmdlet imports reusable text "snippets" from a module or a directory into the current session.
 The snippets are immediately available for use in Windows PowerShell ISE.
-This cmdlet works only in Windows PowerShell® Integrated Scripting Environment (ISE).
+This cmdlet works only in Windows PowerShell? Integrated Scripting Environment (ISE).
```
Comment (zjalexander) [inline reference/3.0/ISE/Import-IseSnippet.md]: looks like the copyright symbol got eaten here

File: reference/3.0/Microsoft.PowerShell.Core/Enable-PSRemoting.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Enable-PSRemoting [-Force] [-SkipNetworkProfileCheck] [-WhatIf] [-Confirm] [<Com
 ## DESCRIPTION
 The **Enable-PSRemoting** cmdlet configures the computer to receive Windows PowerShell remote commands that are sent by using the WS-Management technology.
 
-On Windows Server® 2012, Windows PowerShell remoting is enabled by default.
+On Windows Server? 2012, Windows PowerShell remoting is enabled by default.
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Core/Enable-PSRemoting.md]: copyright symbol eaten

File: reference/3.0/Microsoft.PowerShell.Core/Get-Module.md [hunk 3]
```diff
@@ -251,7 +251,7 @@ PS C:\> Get-Content $m.Path
 GUID="{8FA5064B-8479-4c5c-86EA-0D311FE48875}"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="Â© Microsoft Corporation. All rights reserved."
+Copyright="?? Microsoft Corporation. All rights reserved."
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Core/Get-Module.md]: copyright symbol eaten, also some other weird thing

File: reference/3.0/Microsoft.PowerShell.Core/New-ModuleManifest.md [hunk 4]
```diff
@@ -205,7 +205,7 @@ IscsiTarget                                                 4.0
 Kds                                                         4.0
 Microsoft.PowerShell.Diagnostics                            4.0
 Microsoft.PowerShell.Host                                   4.0
-Microsoft.PowerShell.Management                             4.0â€¦
+Microsoft.PowerShell.Management                             4.0???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Core/New-ModuleManifest.md]: whole bunch of messed up characters here

File: reference/3.0/Microsoft.PowerShell.Core/Receive-PSSession.md [hunk 5]
```diff
@@ -137,7 +137,7 @@ Id Name    ComputerName    State         ConfigurationName     Availability
 
 The third command uses the Invoke-Command cmdlet to run a script in the session in the $s variable.The script begins to run and return data, but a network outage occurs that interrupts the session. The user has to exit the session and restart the local computer.
 PS C:\> Invoke-Command -Session $s -FilePath \\Server12\Scripts\SharedScripts\New-ADResolve.ps1
- Running "New-ADResolve.ps1" â€¦.exit
+ Running "New-ADResolve.ps1" ???.exit
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Core/Receive-PSSession.md]: weird symbols

File: reference/3.0/Microsoft.PowerShell.Core/Receive-PSSession.md [hunk 6]
```diff
@@ -201,7 +201,7 @@ Bug Report - Domain 01
 ----------------------
 ComputerName          BugCount          LastUpdated
 --------------        ---------         ------------
-Server01              121               Friday, December 30, 2011 5:03:34 PMâ€¦
+Server01              121               Friday, December 30, 2011 5:03:34 PM???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Core/Receive-PSSession.md]: weird symbols

File: reference/3.0/Microsoft.PowerShell.Core/Receive-PSSession.md [hunk 7]
```diff
@@ -242,7 +242,7 @@ PS C:\> $j2 = Receive-PSSession -ComputerName Server01 -Name Test
 
 PS C:\> Receive-Job $j
 Return 3
-Return 4â€¦
+Return 4???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Core/Receive-PSSession.md]: weird symbols

File: reference/3.0/Microsoft.PowerShell.Core/Update-Help.md [hunk 8]
```diff
@@ -190,7 +190,7 @@ ADDSDeployment                          en-US
 
 ADFS                                    en-US                                   3.0.0.0
 
-â€¦
+???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Core/Update-Help.md]: weird symbols

File: reference/3.0/Microsoft.PowerShell.Management/Get-WmiObject.md [hunk 9]
```diff
@@ -166,11 +166,11 @@ __SUPERCLASS          : CIM_BIOSElement
 
 __DYNASTY             : CIM_ManagedSystemElement
 
-__RELPATH             : Win32_BIOS.Name="Phoenix ROM BIOS PLUS Version 1.10 â€¦
+__RELPATH             : Win32_BIOS.Name="Phoenix ROM BIOS PLUS Version 1.10 ???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Management/Get-WmiObject.md]: weird characters

File: reference/3.0/Microsoft.PowerShell.Management/Get-WmiObject.md [hunk 10]
```diff
@@ -166,11 +166,11 @@ __SUPERCLASS          : CIM_BIOSElement
...
 
 __PROPERTY_COUNT      : 27
 
-__DERIVATION          : {CIM_BIOSElement, CIM_SoftwareElement, CIM_LogicalElement,â€¦
+__DERIVATION          : {CIM_BIOSElement, CIM_SoftwareElement, CIM_LogicalElement,???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Management/Get-WmiObject.md]: weird characters

File: reference/3.0/Microsoft.PowerShell.Utility/ConvertTo-Csv.md [hunk 11]
```diff
@@ -57,7 +57,7 @@ serProcessorTime","VirtualMemorySize","VirtualMemorySize64","EnableRaisingEvents
 ndardError","WorkingSet","WorkingSet64","Site","Container"
 "Process","powershell","216","597544960","60399616","63197184","21692","C:\WINDOWS\system32\WindowsPowerShell\v1.0\powe
 rshell.exe","Microsoft Corporation","3.4788223","6.1.6587.1 (fbl_srv_powershell(nigels).070711-0102)","6.1.6587.1","Win
-dows PowerShell","Microsoft® Windows® Operating System","8",,"False",,"860","216","5132",".","5636936","Windows PowerSh
+dows PowerShell","Microsoft? Windows? Operating System","8",,"False",,"860","216","5132",".","5636936","Windows PowerSh
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Utility/ConvertTo-Csv.md]: copyright symbol got eaten

File: reference/3.0/Microsoft.PowerShell.Utility/Get-Alias.md [hunk 12]
```diff
@@ -64,7 +64,7 @@ Alias           clc -> Clear-Content
 
 Alias           clear -> Clear-Host
 
-Alias           clhy -> Clear-History â€¦
+Alias           clhy -> Clear-History ???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Utility/Get-Alias.md]: weird characters

File: reference/3.0/Microsoft.PowerShell.Utility/Import-Csv.md [hunk 13]
```diff
@@ -143,7 +143,7 @@ PS C:\> Get-Content .\Links.csv
 113207,about_Aliases113208,about_Arithmetic_Operators113209,about_Arrays113210,about_Assignment_Operators113212, 
 about_Automatic_Variables113213,about_Break113214,about_Command_Precedence113215,about_Command_Syntax144309, 
 about_Comment_Based_Help113216,about_CommonParameters113217,about_Comparison_Operators113218,about_Continue113219, 
-about_Core_Commands113220,about_Data_Sectionâ€¦
+about_Core_Commands113220,about_Data_Section???
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Utility/Import-Csv.md]: weird characters

File: reference/3.0/PSWorkflowUtility/Invoke-AsWorkflow.md [hunk 14]
```diff
@@ -58,7 +58,7 @@ Server03                           279bac28-066a-4646-9497-8fcdcfe9757e    AllSi
 
 localhost                          0d858009-2cc4-47a4-a2e0-da17dc2883d0    RemoteSigned
 
-…
+?
```
Comment (zjalexander) [inline reference/3.0/PSWorkflowUtility/Invoke-AsWorkflow.md]: more encoding nonsense :\

File: reference/4.0/ISE/Import-IseSnippet.md [hunk 15]
```diff
@@ -34,7 +34,7 @@ Import-IseSnippet [-Recurse] -Module <String> [-ListAvailable] [<CommonParameter
 ## DESCRIPTION
 The **Import-IseSnippet** cmdlet imports reusable text "snippets" from a module or a directory into the current session.
 The snippets are immediately available for use in Windows PowerShell ISE.
-This cmdlet works only in Windows PowerShell® Integrated Scripting Environment (ISE).
+This cmdlet works only in Windows PowerShell? Integrated Scripting Environment (ISE).
```
Comment (zjalexander) [inline reference/4.0/ISE/Import-IseSnippet.md]: copyright symbol

File: reference/4.0/Microsoft.PowerShell.Core/Enable-PSRemoting.md [hunk 16]
```diff
@@ -28,7 +28,7 @@ Enable-PSRemoting [-Force] [-SkipNetworkProfileCheck] [-WhatIf] [-Confirm] [<Com
 ## DESCRIPTION
 The **Enable-PSRemoting** cmdlet configures the computer to receive Windows PowerShell remote commands that are sent by using the WS-Management technology.
 
-On Windows Server® 2012, Windows PowerShell remoting is enabled by default.
+On Windows Server? 2012, Windows PowerShell remoting is enabled by default.
```
Comment (zjalexander) [inline reference/4.0/Microsoft.PowerShell.Core/Enable-PSRemoting.md]: copyright symbol

File: reference/4.0/Microsoft.PowerShell.Core/Get-Module.md [hunk 17]
```diff
@@ -272,7 +272,7 @@ PS C:\> Get-Content $m.Path
 GUID="{8FA5064B-8479-4c5c-86EA-0D311FE48875}"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="Â© Microsoft Corporation. All rights reserved."
+Copyright="?? Microsoft Corporation. All rights reserved."
```
Comment (zjalexander) [inline reference/4.0/Microsoft.PowerShell.Core/Get-Module.md]: weird characters

**Comment by saldana** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/PowerShell/PowerShell-Docs, working directory: W:\4p2u-s, target branch: staging, target branch commit: ecd84b0625b900209470b82804e5d50f796c7fa3, source branch: jpjofre-I1188, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C4%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201705042008335168-1190%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request closed without merging**